src/reviews/views.py

```diff
@@ -310,16 +310,31 @@ def review_stages(request):
     ]
 
     if request.method == 'POST':
-        date_time_obj = date_preprocess(
-            DATETIME_INPUT_FORMATS, request.POST['proposals.disable.after'])
-        tz_selectd = pytz.timezone(request.POST['review_timezone'])
-        loc_dt = tz_selectd.localize(date_time_obj).strftime(
-            '%Y-%m-%d %H:%M:%S%z')
 
         for tag in review_stages_var:
             key = settings.CONFERENCE_DEFAULT_SLUG + '.' + tag
             if (tag == 'proposals.disable.after'):
-                value = loc_dt
+                if(request.POST['proposals.disable.after'] == ""):
```

Comment:
Do you want to put `tag` instead of `proposals.disable.after` to void hard coded string here and on line 322? Also, I don't think the parentheses are necessary here.